OpenSSL/crypto.py

```diff
@@ -1520,11 +1519,15 @@ class X509StoreContext(object):
     An X.509 store context.
 
     An :py:class:`X509StoreContext` is used to define some of the criteria for
-    certificate verification.  The information encapsulated in this object
+    certificate verification. The information encapsulated in this object
```

Comment:
*Reallllllly*? That's pretty old school... but fixed nevertheless. :) Will try to keep in mind but that's a habit I had to break a long time ago to escape from school.